src/features/wallet/send/SendLockedField.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Box from '../../../components/Box'
+import Text from '../../../components/Text'
+import InputLock from '../../../assets/images/input-lock.svg'
+
+type Props = {
+  label: string
+  value: string
+  bottom?: boolean
+}
+
+const SendLockedField = ({ label, value, bottom = false }: Props) => {
```

Comment:
This seems generic enough to make it `LockedInput` or something like that. We will need it for transfer too